### PR TITLE
Adding MobyGames external link

### PR DESCRIPTION
**Pull request by Thunderforge**
This gives users a way to learn more about the game with a single click.

Future external links might be added for things like Wikipedia or the ScummVM Wiki.

## Before

(Note that the warning visible in the screenshot was fixed by #154)

image

## After

image

### CODE REVIEW

File: include/OrmObjects/Compatibility.php [hunk 1]
```diff
@@ -59,4 +64,9 @@ public function getDataFiles()
     {
         return $this->getGame()->getDataFiles();
     }
+
+    public function getMobyId()
```
Comment (Mataniko) [inline include/OrmObjects/Compatibility.php:68]: The method is only necessary if we want to expose it to the template

File: include/OrmObjects/Compatibility.php [hunk 2]
```diff
@@ -47,6 +47,11 @@ public function getNotes()
             $notes .= str_replace("- ", "\n- ", parent::getNotes()) . "\n";
         }
 
+        if ($this->getMobyId() != '-1') {
+            $notes .= "\n\n**External Links:**\n";
+            $notes .= "- <a href=\"https://www.mobygames.com/game/{$this->getMobyId()}\">MobyGames</a>\n";
```
Comment (Mataniko) [inline include/OrmObjects/Compatibility.php:52]: Can you use markdown? `[MobyGames](https://www.mobygames.com/game/{$this->getGame()->getMobyId()})`

**Comment by Mataniko** [comment]
Please rebase, as this will re-break unstable platforms

File: include/OrmObjects/Compatibility.php (hunk 1 above)
Comment (Thunderforge) [reply include/OrmObjects/Compatibility.php]: Removed.

File: include/OrmObjects/Compatibility.php (hunk 2 above)
Comment (Thunderforge) [reply include/OrmObjects/Compatibility.php]: I didn't realized that Markdown was supported. I've changed it over to that.

**Comment by Thunderforge** [comment]
@Mataniko Rebased and changed the MobyGames link to Markdown.

**Comment by Mataniko** [comment]
LGTM, thanks for adding this!

**Pull request merged**